pom.xml

```diff
@@ -616,6 +623,12 @@
             </dependency>
 
             <dependency>
+                <groupId>com.microsoft.sqlserver</groupId>
+                <artifactId>mssql-jdbc</artifactId>
+                <version>6.1.0.jre8</version>
```

Comment:
Make sure we know and document what versions of MS SQL Server this driver supports